### ISSUE TITLE
Keeps crashing

### ISSUE THREAD

**Issue by mika4president** [psalm/psalm-vscode-plugin#192]
Issue Type: <b>Bug</b>

# Behaviour

## Expected

XXX

## Actual

XXX

## Steps to reproduce:

[**NOTE**: Self-contained, minimal reproducing code samples are **extremely** helpful and will expedite addressing your issue]

1.

# Diagnostic data

-   PHP version: PHP 8.1.7 (cli) (built: Jun  9 2022 14:21:07) (NTS)
Copyright (c) The PHP Group
Zend Engine v4.1.7, Copyright (c) Zend Technologies
    with Zend OPcache v8.1.7, Copyright (c), by Zend Technologies

-   Psalm version: 4.24.0

<details>

<summary>"Psalm Language Server" channel in the OUTPUT panel (Last 1000 lines)</summary>

<p>

```
[INFO  - 8:43:23 AM] Starting language server
[INFO  - 8:43:24 AM] Psalm Version was detected as 4.24.0
[ERROR  - 8:43:24 AM] Problem parsing /Applications/XAMPP/xamppfiles/htdocs/emerald/psalm.xml:
  Could not resolve config path to /Applications/XAMPP/xamppfiles/htdocs/emerald//boot-lg3-videojs2.php

Problem parsing /Applications/XAMPP/xamppfiles/htdocs/emerald/psalm.xml:
  Could not resolve config path to /Applications/XAMPP/xamppfiles/htdocs/emerald//boot-lg3-videojs2.php

[Info  - 8:43:24 AM] Connection to server got closed. Server will restart.
[INFO  - 8:43:24 AM] Psalm Version was detected as 4.24.0
[ERROR  - 8:43:24 AM] Problem parsing /Applications/XAMPP/xamppfiles/htdocs/emerald/psalm.xml:
  Could not resolve config path to /Applications/XAMPP/xamppfiles/htdocs/emerald//boot-lg3-videojs2.php

Problem parsing /Applications/XAMPP/xamppfiles/htdocs/emerald/psalm.xml:
  Could not resolve config path to /Applications/XAMPP/xamppfiles/htdocs/emerald//boot-lg3-videojs2.php

[Info  - 8:43:24 AM] Connection to server got closed. Server will restart.
[INFO  - 8:43:24 AM] Psalm Version was detected as 4.24.0
[ERROR  - 8:43:24 AM] Problem parsing /Applications/XAMPP/xamppfiles/htdocs/emerald/psalm.xml:
  Could not resolve config path to /Applications/XAMPP/xamppfiles/htdocs/emerald//boot-lg3-videojs2.php

Problem parsing /Applications/XAMPP/xamppfiles/htdocs/emerald/psalm.xml:
  Could not resolve config path to /Applications/XAMPP/xamppfiles/htdocs/emerald//boot-lg3-videojs2.php

[Info  - 8:43:24 AM] Connection to server got closed. Server will restart.
[INFO  - 8:43:24 AM] Psalm Version was detected as 4.24.0
[ERROR  - 8:43:25 AM] Problem parsing /Applications/XAMPP/xamppfiles/htdocs/emerald/psalm.xml:
  Could not resolve config path to /Applications/XAMPP/xamppfiles/htdocs/emerald//boot-lg3-videojs2.php

Problem parsing /Applications/XAMPP/xamppfiles/htdocs/emerald/psalm.xml:
  Could not resolve config path to /Applications/XAMPP/xamppfiles/htdocs/emerald//boot-lg3-videojs2.php

[Info  - 8:43:25 AM] Connection to server got closed. Server will restart.
[INFO  - 8:43:25 AM] Psalm Version was detected as 4.24.0
[ERROR  - 8:43:26 AM] Problem parsing /Applications/XAMPP/xamppfiles/htdocs/emerald/psalm.xml:
  Could not resolve config path to /Applications/XAMPP/xamppfiles/htdocs/emerald//boot-lg3-videojs2.php

Problem parsing /Applications/XAMPP/xamppfiles/htdocs/emerald/psalm.xml:
  Could not resolve config path to /Applications/XAMPP/xamppfiles/htdocs/emerald//boot-lg3-videojs2.php

[Error - 8:43:27 AM] Connection to server got closed. Server will not be restarted.
```

</p>
</details>

<details>

<summary>User Settings</summary>

<p>

```
phpExecutablePath: "/usr/local/bin/php"
phpExecutableArgs: ["-dxdebug.remote_autostart=0","-dxdebug.remote_enable=0","-dxdebug_profiler_enable=0"]
psalmVersion: null
psalmScriptPath: "vendor/vimeo/psalm/psalm-language-server"
psalmScriptArgs: []
disableAutoComplete: false
maxRestartCount: 5
unusedVariableDetection: false
enableVerbose: false
connectToServerWithTcp: false
enableUseIniDefaults: false
logLevel: "INFO"
analyzedFileExtensions: [{"scheme":"file","language":"php"},{"scheme":"untitled","language":"php"}]
configPaths: ["psalm.xml","psalm.xml.dist"]
hideStatusMessageWhenRunning: true
```

</p>
</details>

Extension version: 2.7.0
VS Code version: Code 1.69.2 (3b889b090b5ad5793f524b5d1d39fda662b96a2a, 2022-07-18T16:12:57.074Z)
OS version: Darwin x64 21.3.0
Restricted Mode: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-8279U CPU @ 2.40GHz (8 x 2400)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: enabled<br>video_encode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|2, 3, 3|
|Memory (System)|8.00GB (0.05GB free)|
|Process Argv|--crash-reporter-id 9826536c-7bb8-4111-9825-b425b9428910|
|Screen Reader|no|
|VM|0%|
</details>

**Comment by tm1000** [comment]
You'll need to report this to Psalm
